lib/vnet/vnet_test.go

```diff
@@ -1125,21 +1195,45 @@ func newSelfSignedCA(t *testing.T) tls.Certificate {
 	}
 }
 
-func newServerCert(t *testing.T, ca tls.Certificate, cn string, expires time.Time) tls.Certificate {
-	return newLeafCert(t, ca, cn, expires, x509.ExtKeyUsageServerAuth, cryptosuites.ECDSAP256)
+func newServerCert(
```

Comment:
very useful helpers. should we move them to a common package? we have a few variants lying around too like `api/testhelpers/mtls`, `lib/utils/cert`  😅

Reply:
If helpers stay in `api/testhelpers` and prod code doesn't import from it, they shouldn't be linked to the binary?